### PR TITLE
metadata

### PR DESCRIPTION
**Pull request by anya-ji**

### CODE REVIEW

File: components/uploadButton.js [hunk 1]
```diff
@@ -66,19 +92,23 @@ function UploadButton({ navigation, image }) {
...
-        db.transaction((tx) => {
-          tx.executeSql(
-            "insert into gallery (uri) values (?)",
-            [image.image_uri],
-            () => { },
-            (_, error) => console.log(error)
-          );
-          console.log(image.image_uri);
-        });
+        // db.transaction((tx) => {
```
Comment (tklausner) [inline components/uploadButton.js:95]: should we delete this code? or is there a reason its commented / we want to use it later?

Comment (anya-ji) [reply components/uploadButton.js:95]: Hm this is for the gallery originally... Should I keep it in case we want to implement the gallery at some point?

**Comment by anya-ji** [comment]
- removed redundant code
- added safe area to each view

**Comment by anya-ji** [comment]
- removed redundant code
- added safe area to each view

**Pull request merged**